partcad/src/partcad/wrappers/wrapper_render_threejs.py

```
@@ -0,0 +1,68 @@
+            # 0 means just a triangle
+            result["faces"].append([0, i, j, k])
+
+        json_str = json.dumps(result)
+        with open(path, "w", encoding="utf-8", buffering=256 * 1024) as f:
+            f.write(json_str)
+
+        return {
+            "success": True,
+            "exception": None,
+        }
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)